@xen-orchestra/lite/src/libs/utils.ts

```diff
@@ -60,3 +64,58 @@ export const hasEllipsis = (target: Element | undefined | null) =>
 export function percent(currentValue: number, maxValue: number, precision = 2) {
   return round((currentValue / maxValue) * 100, precision);
 }
+export function getAvgCpuUsage(
+  cpus?: object | any[],
+  opts: { nSequence: number } = { nSequence: 4 }
```

Comment:
@pdonias You can't add a default value inside the TypeScript type declaration.

I think the right syntax is to remove the `opts` parameter to replace it with:

```suggestion
  { nSequence = 4 }: { nSequence?: number } = {}
```

And in this case the type can also be omitted for the same result:

```suggestion
  { nSequence = 4 } = {}
```